applications/callflow/src/cf_exe.erl

```diff
@@ -918,7 +918,7 @@ handle_channel_direct(Call, JObj, Notify) ->
     NewFetchId = kz_json:get_value([<<"Custom-Channel-Vars">>, <<"Fetch-ID">>], JObj),
     case OrgFetchId =:= NewFetchId of
         'true' ->
-            ReplacedBy = kz_json:get_value(<<"Connecting-Leg-A-UUID">>, JObj),
+            ReplacedBy = kz_json:get_value(<<"Connecting-Leg-B-UUID">>, JObj),
```

Comment:
You should add this to kz_call_event.erl and update this function's spec. Maybe even rename JObj to CallEvent or something descriptive :)